### PR TITLE
feat: subtasks new design and features

### PR DESCRIPTION
**Pull request by juniusfree**
**Demo:** https://www.loom.com/share/034291e609824b27a48be39e515761e4

related backend PR https://github.com/wondrous-dev/wondrous-backend/pull/456

tasks: https://app.wonderverse.xyz/organization/wonderverse/boards?task=61446546393138051

### CODE REVIEW

File: wondrous-app/components/Common/TaskSubtask/TaskSubtaskList/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    margin: 0;
+    svg {
+      path {
+        stroke: ${({ theme }) => theme.palette.white};
+      }
+    }
+  }
+`;
+
+export const TaskSubtaskSafeImage = styled(SafeImage)``;
```
Comment (Lamperoyge) [inline wondrous-app/components/Common/TaskSubtask/TaskSubtaskList/styles.tsx:97]: if we don't override the SafeImage I think we can import SafeImage directly to TaskSubtaskList

File: wondrous-app/components/Common/TaskSubtask/TaskSubtaskList/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    align-items: center;
+    background: #171717;
+    border-radius: 6px;
+    display: flex;
+    height: 68px;
+    justify-content: center;
+  }
+`;
+
+export const SubtaskTaskListHasMore = styled.div``;
```
Comment (Lamperoyge) [inline wondrous-app/components/Common/TaskSubtask/TaskSubtaskList/styles.tsx:123]: Same here I think we can use a basic div in the TaskSubtaskList

**Review by Lamperoyge — COMMENTED** [review]
left few minor comments but overall looks great!

**Comment by juniusfree** [comment]
> left few minor comments but overall looks great!

@Lamperoyge Thanks for the feedback. The PR is updated.

**Pull request merged**